packages/core/src/components/Tipseen/Tipseen.tsx

```diff
@@ -131,20 +152,23 @@ const Tipseen: VibeComponent<TipseenProps> & {
             <IconButton
               hideTooltip
               className={cx(styles.tipseenCloseButton, {
-                [styles.dark]: closeButtonTheme === TipseenCloseButtonTheme.DARK
+                [styles.dark]:
+                  closeButtonTheme === TipseenCloseButtonTheme.DARK ||
+                  closeButtonTheme === TipseenCloseButtonTheme.FIXED_DARK
               })}
               onClick={onClose}
               size={Button.sizes.XS}
               kind={Button.kinds.TERTIARY}
-              color={Button.colors.ON_PRIMARY_COLOR}
+              // @ts-ignore
+              color={closeButtonColor}
               ariaLabel={overrideCloseAriaLabel}
               icon={CloseSmall}
             />
           )}
           <TipseenTitle text={title} className={cx(styles.tipseenTitle, titleClassName)} />
         </div>
-        <Text color={Text.colors.ON_PRIMARY} type={Text.types.TEXT2} element="p" className={cx(styles.tipseenContent)}>
-          {content}
+        <Text color={textColor} type={Text.types.TEXT2} element="p" className={cx(styles.tipseenContent)}>
+          <TipseenContext.Provider value={color}>{content}</TipseenContext.Provider>
```

Comment:
I think it will mess up the tooltip if it's present, worth checking (since it expects just a simple text)